tests/integration/registration/test_inference.py

```diff
@@ -0,0 +1,79 @@
+import pytest
+from utils.data_source_utils import (
+    prep_file_source,
+    simple_bq_source_using_query_arg,
+    simple_bq_source_using_table_ref_arg,
+)
+
+from feast import Entity, RepoConfig, ValueType
+from feast.errors import RegistryInferenceFailure
+from feast.feature_view import FeatureView
+from feast.inference import (
+    update_data_sources_with_inferred_event_timestamp_col,
+    update_entities_with_inferred_types_from_feature_views,
+)
+
+
+def test_update_entities_with_inferred_types_from_feature_views(
+    simple_dataset_1, simple_dataset_2
```

Comment:
Is there still any reason to use this `simple_dataset_1/2` if we have the universal repo?